tools/nuttx-gdbinit

```diff
@@ -406,36 +397,31 @@ define _switch_tcb_simx86
 end
 
 define _restore_current_tcb
-  # TODO: SMP
-  set $tcb = g_readytorun->head
-  _switch_tcb $tcb
+  _examine_target
+  _running_task
+  _switch_tcb $rtcb
   set $_current_tcb = 0x0
 end
 
 define nxthread
-  _examine_target
-  _save_current_tcb
-  set $hash = ($arg0 & ($_target_max_tasks - 1))
+  set $hash = ($arg0 & (g_npidhash - 1))
   set $tcb = g_pidhash[$hash]
   if ($tcb != 0x0)
+    _switch_tcb $tcb
     _print_thread $tcb
-    if ($argc == 1)
-      _switch_tcb $tcb
-    end
     if ($argc == 2)
       if ($arg1 == 1)
-	_switch_tcb $tcb
-	where
+        _switch_tcb $tcb
+        where
       end
     end
   end
-  _restore_current_tcb
 end
 
 define nxthread_all_bt
```

Comment:
remove nxthread_all_bt, we can use the standard command "thread apply all bt"

Reply:
"thread apply all bt" could not be redefined